2227/2/lib/rapper_analizator.rb

```diff
@@ -0,0 +1,52 @@
+# Analysis of rappers statistic
+class RapperAnalyzer
+  DEFAULT_ROUNDS_COUNT = 3
+
+  def self.calculate_statistic(name)
+    new(name).fetch_statistic
+  end
+
+  def initialize(name)
+    all_battles = DataStorage.find_all_battles(name)
+    @number_of_battles = all_battles.size
+    @battles_text = all_battles.values.join(' ')
+  end
+
+  def fetch_statistic
```

Comment:
I would prefer to see separate methods for every column. And if u need exactly this data structure somewhere then you always can create it in place